src/index.html

```diff
@@ -11,7 +11,198 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="field-group">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          How old are you?
+
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+
+          <input
+            type="date"
+            name="dateOfBirth"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+
+          <input
+            type="checkbox"
+            name="agreement"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field-group">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+
+          <input
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="20"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field-group">
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <div class="form-field">
+          Do you love cats?
+
+          <label>
+
+            <input
+							type="radio"
+							name="loveForCats"
+							value="yes"
+            >
```

Comment:
```suggestion
            <input
              type="radio"
              name="loveForCats"
              value="yes"
            >
```
Ну аж дуже багато відступів

Reply:
очень странно, в документе нормально отображается((